Automattic/newspack-theme
# Style C: Add option for 'Style 2', and update header and body fonts

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a style pack option for 'Style 2' (Style C in the mockups).

It also enqueues the fonts from that mockup (Montserrat) and sets the header and body fonts to match the design.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`; confirm there are no errors (since the build process has been changed). 
2. Navigate to the Customizer > Style Pack; switch to 'Style 2'.
3. On the front-end, confirm that it's now loading the 'Style 2' fonts -- Montserrat for the headers, and sans-serif for the body fonts:

![image](https://user-images.githubusercontent.com/177561/62719878-c36a6c80-b9bd-11e9-90a3-926bf666acd4.png)

### Other information:
